### PR TITLE
Improve webhooks

### PR DESCRIPTION
**Pull request by LordJavi**
Attempt to increase parser speed avoiding manage some webhooks

### CODE REVIEW

File: src/services/consumer.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ class Consumer {
                                 continue;
                             }
                             const gym = Gym.fromFort(fort.cell, fort.data);
-                            await gym.triggerWebhook();
+                            gym.triggerWebhook();
```
Comment (versx) [inline src/services/consumer.js:60]:
```suggestion
                            await gym.triggerWebhook();
```
We need await for promises

File: src/services/consumer.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ class Consumer {
                                 continue;
                             }
                             const pokestop = Pokestop.fromFort(fort.cell, fort.data);
-                            await pokestop.triggerWebhook(false);
+                            pokestop.triggerWebhook(false);
```
Comment (versx) [inline src/services/consumer.js:74]:
```suggestion
                            await pokestop.triggerWebhook(false);
```
We need await for promises

File: src/services/consumer.js [hunk 3]
```diff
@@ -373,7 +373,7 @@ class Consumer {
                 let conditions = weather[i];
                 try {
                     const weather = Weather.fromClientWeather(conditions.cell.toString(), conditions.data, ts);
-                    await weather.triggerWebhook();
+                    weather.triggerWebhook();
```
Comment (versx) [inline src/services/consumer.js:376]:
```suggestion
                    await weather.triggerWebhook();
```
We need await for promises

File: src/models/gym.js [hunk 4]
```diff
@@ -84,6 +85,10 @@ class Gym extends Model {
      * trigger webhooks
      */
     async triggerWebhook() {
+        if (!config.webhooks.enabled || config.urls.length === 0) {
```
Comment (Mygod) [inline src/models/gym.js:88]: Should probably move this to a common function?

**Comment by versx** [comment]
Uhh, works? 😅

**Comment by Mygod** [comment]
Sure I guess.

**Pull request merged**